test/test.html

```diff
@@ -44,6 +44,7 @@
     <script src="/test/modules/rectangle.js"></script>
     <script src="/test/modules/ajax-tiles.js"></script>
     <script src="/test/modules/imageloader.js"></script>
+    <script src="/test/modules/iiif.js"></script>
```

Comment:
Please add this to test/coverage.html as well.

Reply:
Done.